lib/facter/winlogbeat_version.rb

```diff
@@ -0,0 +1,10 @@
+require 'facter'
+Facter.add('winlogbeat_version') do
+  confine :kernel => %w(Windows) # rubocop:disable Style/HashSyntax
+  if File.exist?('c:\Program Files\Winlogbeat\winlogbeat.exe')
```

Comment:
no way to fix this.... Worst case the fact does not work and you can manually specify the major_version you want to get the right config file format.